[warning] stb_image_resize2.h

> In file included from rtextures.c:216:
> external/stb_image_resize2.h:2866:14: warning: ‘stbir__support_zero’ defined but not used [-Wunused-function]
>  2866 | static float stbir__support_zero(float s, void * user_data)
>       |              ^~~~~~~~~~~~~~~~~~~
> external/stb_image_resize2.h:2503:13: warning: ‘stbir_simd_memcpy’ defined but not used [-Wunused-function]
>  2503 | static void stbir_simd_memcpy( void * dest, void const * src, size_t bytes )
>       |             ^~~~~~~~~~~~~~~~~
> ar rcs ../src/libraylib.a rcore.o rshapes.o rtextures.o rtext.o utils.o rglfw.o rmodels.o raudio.o
> raylib static library generated (libraylib.a) in ../src!
>

--- a/src/rtextures.c
+++ b/src/rtextures.c
@@ -212,8 +212,18 @@
 
 #define STBIR_MALLOC(size,c) ((void)(c), RL_MALLOC(size))
 #define STBIR_FREE(ptr,c) ((void)(c), RL_FREE(ptr))
+
+#if defined(__GNUC__) // GCC and Clang
+    #pragma GCC diagnostic push
+    #pragma GCC diagnostic ignored "-Wunused-function"
+#endif
+
 #define STB_IMAGE_RESIZE_IMPLEMENTATION
 #include "external/stb_image_resize2.h"     // Required for: stbir_resize_uint8_linear() [ImageResize()]
+
+#if defined(__GNUC__) // GCC and Clang
+    #pragma GCC diagnostic pop
+#endif
 
 #if defined(SUPPORT_FILEFORMAT_SVG)
     #define NANOSVG_IMPLEMENTATION          // Expands implementation

